rules/library.bzl

```diff
@@ -684,17 +684,6 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
         import_vfsoverlays.append(import_name + "_vfs")
 
     for xcframework in kwargs.pop("vendored_xcframeworks", []):
-        import_headers = native.glob(
-            ["**/*.h"],
-        )
-        import_module_maps = native.glob(
-            ["**/*.modulemap"],
-        )
-        if len(import_module_maps) > 0:
-            import_module_map = import_module_maps[0]
-        else:
-            import_module_map = None
-        vfs_imported_framework = _find_imported_framework_name(import_headers)
```

Comment:
None of this was used for the XCFramework case, most of the logic is handled in the `_xcframework` function.